README.md

```diff
@@ -144,6 +144,7 @@ Please also see the [example configs](./docs/examples/) for special cases.
 
 - [`selector-nest-combinators`](./src/rules/selector-nest-combinators/README.md): Require or disallow nesting of combinators in selectors.
 - [`selector-no-redundant-nesting-selector`](./src/rules/selector-no-redundant-nesting-selector/README.md): Disallow redundant nesting selectors (`&`).
+- [`selector-no-union-class-name`](./src/rules/selector-no-union-class-name/README.md): Disallow union class name with `&`.
```

Comment:
The description here and in the rule readme could be something like ` Disallow union class names with the parent selector (&)` . What do you think? 

We have been using the term "nesting selector" before, but the official one seems to be "parent selector":
https://sass-lang.com/documentation/style-rules/parent-selector

Reply:
Yours seems better than mine!

Reply:
FIxed!: af5cd12